src/components/CookiesDialog.vue

```diff
@@ -0,0 +1,101 @@
+<!-- eslint-disable vue-i18n/no-raw-text -->
+<template>
+  <div
+    class="cookies-dialog"
+    @click.stop
+  >
+    <img src="../assets/cookieImg.svg">
+    <div class="info">
+      Superhero.com respects your privacy. Third-party tracking cookies are blocked by default.
+      In order to play content such as video or audio
+      you have to allow third-party cookies by content provider.
+    </div>
+    <div>
+      <button
+        class="cookies-button"
+        @click="allow"
+      >
+        Allow {{ scope }}
+      </button>
+      <RouterLink
+        v-if="address"
+        class="cookies-button"
+        :to="`/user-profile/${address}`"
+      >
+        Customize policy
+      </RouterLink>
+    </div>
+  </div>
+</template>
+
+<script>
+import backendAuthMixin from '../utils/backendAuthMixin';
+
+export default {
+  mixins: [backendAuthMixin()],
+  props: { address: { type: String, default: '' }, scope: { type: String, required: true } },
+  methods: {
+    async allow() {
+      if (this.address) {
+        await this.backendAuth(`setCookies${this.scope}`, {
+          scope: this.scope,
+          status: 'ALLOWED',
+        });
+      }
+      this.$store.commit('setCookiesListEntry', {
+        scope: this.scope,
+        status: 'ALLOWED',
+      });
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.cookies-dialog {
+  position: absolute;
+  z-index: 9;
+  left: -4%;
+  background-color: $buttons_background;
+  border: 1px solid $article_content_color;
+  border-radius: 0.5rem;
+  width: 108%;
```

Comment:
I can't get the designs. It is not modal, it is shown just instead of content, but why we need to make it absolute positioned and stretch out of the row? Can somebody share with me the link to the designs?